### ISSUE TITLE
Issues in Chapter03, solution to  exercise 3.14.18 (e)

### ISSUE THREAD

**Issue by TebooNok** [telmo-correa/all-of-statistics#11]
[https://github.com/telmo-correa/all-of-statistics/blob/master/Chapter%2003%20-%20Random%20Variables.ipynb](chapter3)
Hi, there might be an issue with Chapter03, on the solution to 
In equation (13), it shows : 1-P(|X|<c)=1-P(-c<X<-c), it should be 1-P(-c<X<c).
Then in equation (14), it should be 1-P((-c-3)/4<Z<(c-3)/4), because it may not be symmetric in the normal distribution.
So finally I got P(|X|>|x|)=Phi((x+3)/4)-Phi((x-3)/4)=0.005, and using brute-force finding that the solution of |x| is 9.5464.
But surely I don't know if I was right.

**Comment by Denisoidd** [comment]
Yes, I agree with @TebooNok. I think authors put minus before whole expression -(c-3)/4 instead of only -c. Which resulted in obtaining Phi function with symmetric arguments: Phi(-(c-3)/4) and Phi((c-3)/4). I think here brut-force is the only solution.

**Comment by TASPlasma** [comment]
Agreed, and it can be verified that plugging in $c~=10.8399$ back into the original $1-\mathbb{P}\left(X \in (-c, c)\right)$ does not yield 0.05, but rather something closer to 0.025. The 9.5464 value is roughly what I obtained by brute force, which I think is unavoidable for the problem.
